### PR TITLE
[3.x] Modernisation

### PR DESCRIPTION
**Pull request by clemblanco**
* No functionality change
* Linting
* Minor modernisations
* Moved from Travis CI to Github Actions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,3 +44,19 @@ If you're using Lumen, add the following line to your `bootstrap/app.php` file:
...
 $app->register(PodPoint\I18n\Providers\CountriesServiceProvider::class);
 ```
+
+## Changelog
+
+Please see [CHANGELOG](CHANGELOG.md) for more information on what has changed recently.
+
+## License
+
+The MIT License (MIT). Please see [License File](LICENCE.md) for more information.
```
Comment (willthorpe) [inline README.md:54]: Licence not License here

license is the verb for the UK whereas America uses license for both :D I've learnt something while looking at this: https://www.grammar-monster.com/easily_confused/licence_license.htm

File: README.md [hunk 2]
```diff
@@ -14,15 +14,24 @@ The countries are indexed using their uppercase ISO codes (alpha2/cca2).
...
+```bash
+composer require pod-point/countries:^4.0
+```
+
+For Laravel 7.x or 8.x and PHP >= 7.3
+```bash
+composer require pod-point/countries:^5.0
+```
+
+Then finally, if you're using a Laravel version which doesn't support Auto Discovery, add the service provider to your `config/app.php` providers array:
```
Comment (willthorpe) [inline README.md:34]: Then, finally, if

File: composer.json [hunk 3]
```diff
@@ -1,7 +1,15 @@
 {
     "name": "pod-point/i18n",
     "description": "A package of useful components for internationalisation in Laravel applications.",
-    "type": "library",
+    "keywords": ["laravel", "i18n", "internationalisation", "tax", "currencies", "markets", "languages", "countries"],
```
Comment (willthorpe) [inline composer.json:4]: Is this package doing too much? 😆

File: .github/workflows/run-tests.yml [hunk 4]
```diff
@@ -0,0 +1,41 @@
+name: run-tests
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
```
Comment (willthorpe) [inline .github/workflows/run-tests.yml:7]: Only thing here is do our servers use ubuntu 18.04?

**Review by willthorpe — APPROVED** [review]
Just a couple of minor grammar things.

File: .github/workflows/run-tests.yml (hunk 4 above)
Comment (clemblanco) [reply .github/workflows/run-tests.yml:7]: No we're using 16.04 but it's becoming deprecated and also it's a public package. Doesn't really matter much.

File: composer.json (hunk 3 above)
Comment (clemblanco) [reply composer.json:4]: Yes and no?

**Review by RichToms — APPROVED** [review]

**Pull request merged**